src/stirling/source_connectors/tcp_stats/print_utils.h

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
This file (`print_utils.h`) may need to live in a branch on your side until we figure out how to connect this to a json output or a different way of getting the data out.

Reply:
Sure, reverted.